themes/dark/assets/js/lib/vuex.js

```diff
@@ -0,0 +1,938 @@
+/**
```

Comment:
Looks like this was committed by mistake? None of the other themes have this file, and it's not imported in `main.mako`.